assets/style/style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  margin: 0;
+}
+
+.nav-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 24px;
+}
```

Comment:
- [ ] Your design 👇 
![userToolbar](https://user-images.githubusercontent.com/78251927/204957857-d1c0452f-5ce1-4f76-baed-8a371aab7289.PNG)

- [ ] Figma design 👇 
![FigmaTtoolbar](https://user-images.githubusercontent.com/78251927/204957852-16f87f8b-7928-4a8a-8549-331e7e0d99c9.PNG)

- [ ] Please kindly add the section margins (top, bottom, left, right) as it is defined in the design

Reply:
Hi @abdulhamiid, thank you for all of your advice. But, may I ask you how did you open my to see it in the browser, please? I don't know if i am mistaken, but let me show you how i see it in my browser.
![Screenshot 2022-12-01 083259](https://user-images.githubusercontent.com/28353673/205053987-d52d26f8-84a7-4329-9567-a4b6319e9575.png)

Reply:
Yes I see but if I change the screen size, the error still occurs
### 1 (using the mobile icon)
![p11](https://user-images.githubusercontent.com/78251927/205106292-09d2c98b-0d2d-4c25-846b-e653ef22711f.png)

### 2 (without using the icon)

![p22](https://user-images.githubusercontent.com/78251927/205106344-b570908f-ec22-4f7c-a578-979ae45975f4.png)

Reply:
Thank you @abdulhamiid 
I will solve it.
I didn't know how to reproduce the error so I am not sure it is solved in my recent PR.
Again, thanks.

Reply:
I fix it, Thanks @abdulhamiid .✅